.github/workflows/mdbook_build.yaml

```diff
@@ -32,4 +32,4 @@ jobs:
         run: |
           cd book
           mdbook build
-          cd ..
\ No newline at end of file
+          cd ..
```

Comment:
nit: why is this necessary?

Reply:
it probably isn't. Feel free to remove if you want

Reply:
`mdbook build` just has to happen from the `book` dir